### ISSUE TITLE
Making a payment/marking invoice as paid

### ISSUE THREAD

**Issue by t2thec** [amochohan/xerolaravel#4]
Hi ya,

I've been trying to mark and invoice as paid. from reading, I believe that you can't simply mark the invoice's status as 'Paid' as the readme suggests as Xero needs to have a payment associated with it. Do you have any experience with this?

I've added Payments to the Service Provider and I am creating a Payment object, however, going through the API, I can't see how you would make that payment. There is no `setPayments` on the invoice model and I can't seem to just save the payment on the xero object: `$xero->save($payment);`

Any ideas?

**Comment by trevorgehman** [comment]
You are correct. You need to create a payment for the invoice. Unfortunately, the service provider included here does not setup Facades for `Payments` and `Accounts`, which you need. Still you can accomplish this:

```
$xero = App::make('XeroPrivate');
$invoice = App::make('XeroInvoice');
$invoice->setInvoiceID(....);
$account = new \XeroPHP\Models\Accounting\Account();
$account->setCode(....);
$xero_payment = new \XeroPHP\Models\Accounting\Payment();
$xero_payment->setAccount($account);
$xero_payment->setInvoice($invoice);
$xero_payment->setAmount(....);
$xero->save($xero_payment);
```

A good improvement would be if the `XeroServiceProvider` added `XeroPayment` and `XeroAccount` to the IoC bindings.

**Comment by amochohan** [comment]
Solved with #8
